docs/framework/contributing/changelog-entries.md

```diff
@@ -0,0 +1,318 @@
+
+Changes affect one or more packages. List the package that was most impacted by the change first.
+
+However, it is possible to skip this part if many packages are affected. This usually indicates a generic change. In this case, having all the packages listed would reduce the changelog's readability.
+
+The package name is based on the npm package name, but the `@ckeditor/` prefix is stripped.
+
+If your change is related to the main package, use `ckeditor5` as the package name.
+
+<info-box>
+    If the commit introduces a breaking change across the entire project (a generic change), you do not need to specify the package name.
+</info-box>
+
+### Referencing issues
+
+When creating PRs that address specific issues, use the following messages to indicate them.
+
+* `Closes` &ndash; When the PR resolves an issue.
+* `See` &ndash; When the PR references an issue but has not resolved it yet.
+
+Both fields (`closes` and `see`) can contain multiple references, but they must follow the same format:
+
+* `14724` &ndash; A simple issue number.
+* `ckeditor/ckeditor5#14724` &ndash; A full reference to an issue in the CKEditor 5 repository.
+* `https://github.com/ckeditor/ckeditor5/issues/14724` &ndash; A full URL to an issue in the CKEditor 5 repository.
+
+### Giving credit
+
+When closing a non-core contributor's PR, add information about the contributor to the changelog entry file using the `communityCredits` field. It should contain a list of GitHub usernames of contributors who should be credited for this change.
+
+### Description
+
+Write a concise and meaningful summary of the change. This main message will appear in the public changelog, so keep it clear, user-facing, and relevant.
+
+Use the `ClassName#methodName()` format when referencing methods. This ensures consistency across all entries.
+
+**Example:**
+
+```
+MarkerCollection#has()
+```
+
+You may include multiple sentences if additional context is helpful.
+
+### Examples of correct entry formatting
+
+<info-box>
+    Unlike the previous Git-based system, which captured all commit types, including internal changes, the new file-based changelog focuses exclusively on public, user-facing changes, ensuring the final changelog remains clear and relevant to end users.
+</info-box>
+
+A new feature without any breaking changes.
+
+```md
+---
+type: Feature
+scope:
+- ckeditor5-ui
+  closes:
+- 1
+---
+
+Added support for RTL languages.
+
+RTL content will now be rendered correctly.
+```
+
+A generic bug fix for an existing feature that affects many packages (closes two tickets):
+
+```md
+---
+type: Fix
+closes:
+   - 2
+   - 3
+---
+
+The editor will be great again.
+```
+
+An improvement that is not backward compatible and sent by a non-core contributor. Public API was changed:
+
+```md
+---
+type: Other
+scope:
+- ckeditor5-utils
+closes:
+- 9
+---
+
+Extracted the `utils#foo()` to a separate package.
+```
+
+```md
+---
+type: Feature
+scope:
+    - ckeditor5-engine
+closes:
+    - 9
```

Comment:
In these examples, there are sometimes 4 spaces, sometimes 3 spaces, sometimes no spaces.

Reply:
Fixed in b1bf4f4.